components/automate-chef-io/content/docs/reports.md

```diff
@@ -72,6 +72,11 @@ Please note the only supported filters for deep filtering are:
     * one profile
     * one profile and one of its child controls
 
+### Waivers
+
+Waived status of nodes can be seen throughout the pages where status is listed. Users can hover over waived controls' status icon in the controls list to view more details about the waiver applied to the control.
```

Comment:
@vjeffrey When you describe the "waived controls' status icon in the controls list", does that refer to the Node Status column in the Controls list (/compliance/reports/controls) OR are you talking about the Waived Nodes filter in the Node list (compliance/reports/nodes)? Both lists are located under Compliance Reports.

Reply:
Node Status column in the Controls list